helm/external-dns-app/templates/deployment.yaml

```diff
@@ -214,8 +222,6 @@ spec:
       {{- if or .Values.secretConfiguration.enabled .Values.extraVolumes (eq .Values.provider "azure") }}
       volumes:
         {{- if eq .Values.provider "azure" }}
-        - name: config
-          emptyDir: {}
```

Comment:
are you sure this is intended? wouldn't it break vintage?

Reply:
for more context, there is a initContainer copying  `/etc/kubernetes/config/azure.yaml` to `/config/azure.yaml`. I guess it's there to avoid exposing the entire `/etc/kubernetes` to the container